### DISCUSSION TITLE
[BUG] : Start failed

### DISCUSSION THREAD

**Discussion by webislands** [sinshu/managed-doom#26] [General]
i am start the game is not run 

![Sans titre](https://user-images.githubusercontent.com/11302314/199882392-89472dd9-3466-4e95-b216-19979da7acdf.png)

**Comment by sinshu** [comment]
Did you get any error messages that looked like this?

![image](https://user-images.githubusercontent.com/1734672/199970443-39b580fd-c06c-4b81-bfaf-740b0a268d87.png)

**Reply by webislands** [reply]
nothing just wait many minute

**Comment by webislands** [comment]
i am dl lasted version version do you need more file i am don see in folder wad and another file ???

**Reply by sinshu** [reply]
If you don't have any IWADs in the directory, an error message will show up like I mentioned before.
"Nothing happens" should not happen...

Anyway, you need some IWAD to execute this program.
If you don't own any Doom games, you can download the shareware version (doom1.wad) from here.
https://doomwiki.org/wiki/DOOM1.WAD

**Reply by webislands** [reply]
sorry i am french what version of csharp 4 or 5 6,,, and vs studion 2022 , 2019 ,... i am try on windows 10

**Reply by webislands** [reply]
put iwad in directory notting work  and put in c:\managed-doom nothing sorry

**Reply by sinshu** [reply]
Here are my settings:
- OS: Windows 10
- IDE: Visual Studio 2022
- Target framework: .NET 6
- Language version: 10.0

**Reply by webislands** [reply]
same do you use nuget for add to the project ? and wad pin in same directory

**Reply by sinshu** [reply]
Yes, I use NuGet.
Are you trying to compile the program?

**Comment by webislands** [comment]
i  am compile and add wad in debug folder and same bug

**Comment by webislands** [comment]
is ok a i am clean all filles and remove in all in visual studio reload git and is york i am down understant is work thank for your time

all is work i am dont understand

thank for support
